api/SVGGraphicsElement.json

```diff
@@ -0,0 +1,243 @@
+{
+  "api": {
+    "SVGGraphicsElement": {
+      "__compat": {
+        "mdn_url": "https://developer.mozilla.org/docs/Web/API/SVGGraphicsElement",
+        "support": {
+          "webview_android": {
+            "version_added": null
+          },
+          "chrome": {
+            "version_added": true
+          },
+          "chrome_android": {
+            "version_added": true
+          },
+          "edge": {
+            "version_added": true
+          },
+          "edge_mobile": {
+            "version_added": true
+          },
+          "firefox": {
+            "version_added": true
+          },
+          "firefox_android": {
+            "version_added": true
```

Comment:
There is a note associated with Firefox Mobile, that I think should be reflected here.